File: Leetcode/944. Delete Columns to Make Sorted/solution.py
class Solution:
    def minDeletionSize(self, strs: List[str]) -> int:
        delete = 0
        for char in range(len(strs[0])):
            for word in range(1, len(strs)):
                if not strs[word-1][char] <= strs[word][char]:
                    delete += 1
                    break
        return delete

File: Leetcode/944. Delete Columns to Make Sorted/solution.rs
impl Solution {
    pub fn min_deletion_size(strs: Vec<String>) -> i32 {
        let mut delete = 0;
        for col in 0..strs[0].len() {
            for row in 1..strs.len() {
                if !(strs[row-1].as_bytes()[col] <= strs[row].as_bytes()[col]) {
                    delete += 1;
                    break;
                }
            }
        }
        delete
    }
}